s3tests/functional/test_s3.py

```diff
@@ -3272,9 +3273,10 @@ def test_bucket_create_naming_good_long_254():
 @attr(method='put')
 @attr(operation='create w/255 byte name')
 @attr(assertion='fails with subdomain')
-def test_bucket_create_naming_good_long_255():
-    _test_bucket_create_naming_good_long(255)
+def test_bucket_create_naming_bad_long_255():
+    _test_bucket_create_naming_bad_long(255)
 
+'''
```

Comment:
If this test is no longer valid can you remove it?

Reply:
Ok. Done in a 2nd commit.